### ISSUE TITLE
Migrate from MCMCDiagnostics.jl to MCMCDiagnosticTools.jl

### ISSUE THREAD

**Issue by davidanthoff** [raddleverse/MimiBRICK.jl#83]
MCMCDiagnostics.jl is no longer maintained and now outputs a warning whenever one loads it (very annoying). The suggestion is that one uses https://turinglang.org/MCMCDiagnosticTools.jl/stable/ instead, so we should migrate to that.

**Comment by lrennels** [comment]
@tonyewong any thoughts on this, I can help, it's jus taking MimiGIVE irritatingly chatty :)

**Comment by tonyewong** [comment]
My thoughts are: we should update this. I'll have a look this afternoon/tomorrow!

**Comment by tonyewong** [comment]
Done - https://github.com/raddleverse/MimiBRICK.jl/commit/eb90106f93143bfe2edf25895651994f7ed27c2d

**Comment by lukem12345** [comment]
Is it possible to tag a new release of MimiBRICK? I believe that this MCMDiagnostics dependency could force a lower version of `StatsBase` in some downstream use cases.

**Comment by lrennels** [comment]
@davidanthoff @tonyewong I think it's reasonable to tag a new patch or minor release here right, ie. v1.0.2 or v1.1.0? MimiCIAM will then pull in the new MimiBRICK, since it has MimiBRICK as a dependency but for any version with major version 1.

I think as far as I can tell the correct flow is below, only #1 is needed for now and would get things working for @lukem12345 

1. As @lukem12345 requested tag a new release of MimiBRICK, either `v1.1.0` or `v1.0.2`
2. Tag a new release of MimiCIAM -- a _major_ one because some underlying data has changed -- so that would be `v2.0.0`
3. We can decide when we want to update MimiGIVE to depend on the newer version of MimiCIAM

**Comment by tonyewong** [comment]
Yes, this is a good idea. I have to dig back in but I apparnetly basically set everything up to deal with the MCMCDiagnostics.jl vs MCMCDiagnosticTools.jl issue ages ago and then got distracted. Step 1 of @lrennels 's suggestion should be quick and easy, and I can knock that out this week! The other ones we can chat and make sure we're on the same page about what updates to pack into that.
